### ISSUE TITLE
[Feature request] Add manual download button

### ISSUE THREAD

**Issue by mylogon341** [meeb/tubesync#384]
Tl;dr
Add a manual download button

With bigger or older yt channels, they might have several hundred videos. When adding them you don't always want all the videos (especially if you're a long time viewer) so when adding the source, you set the download threshold to 1 week. However, you look through their backlog (in the skipped media) and see some videos that you enjoyed in the past and want to fetch ad-hoc. There is currently no way to do this (other than click on the yt link) as the 1 week cut off date in tubesync is gospel.
Can you add a 'manual override' button that trumps other checks and just kicks off a download? That'd be nice, I think.

**Comment by meeb** [comment]
Thanks for the issue. Sounds like a reasonable suggestion, I'll pop it on the wishlist.

**Comment by chminsc** [comment]
where to set the cufoff date ? 
i didnt find that option. thanks

**Comment by tcely** [comment]
Is there a better way to do this than adding a manual download column to the `Media

**Comment by meeb** [comment]
Would just firing off a download task for the item when you click a button do it? It'll set the various downloaded flags upon download completion.

**Comment by tcely** [comment]
That would not allow bypassing the source settings, skip, etc.

**Comment by tcely** [comment]
There has been some improvement for this issue.

You can override the checklist that would normally prevent the download task from attempting a manually initiated download.

This avoids the column that I was considering adding to the media table earlier in this issue.

https://github.com/meeb/tubesync/blob/4dc7243972e9b999053eae1d29c4e07a575f0960/tubesync/sync/tasks.py#L643-L664

If #1037 is being worked on (👋 @Anusien), this should make that work easier than it was before. Avoiding database migration is my new goal with my recent experiences. 🙃
